src/main/webapp/index.html

```diff
@@ -35,39 +35,42 @@ <h1 class="index-registration-block-child">TravIS: Travel Information System</h1
         <form action="/calculate-trip" method="POST" id="startTripForm">
           <div class="form-row">
             <div class="form-group col-md-3">
-              <input type="text" placeholder="Destination" class="form-control" 
-                name="inputDestination" id="inputDestination">
+              <label for="inputDestination">Destination</label>
+              <input type="text" placeholder="Where to?" class="form-control" 
+                name="inputDestination" id="inputDestination" 
+                onchange="checkValidInput('inputDestination'); checkNextButton();
+                  checkSubmitButton()">
             </div>
             <div class="form-group col-md-3">
-              <input type="text" placeholder="Start Date" class="form-control" 
-                name="inputStartDate" id="inputStartDate">
+              <label for="inputStartDate">Start Date</label>
+              <input type="date" class="form-control" name="inputStartDate" 
+                id="inputStartDate" oninput="setEndDateValue()"
+                onchange="checkValidInput('inputStartDate'); checkNextButton(); 
+                  checkSubmitButton()">
             </div>
             <div class="form-group col-md-3">
-              <input type="text" placeholder="End Date" class="form-control is-valid" 
-                name="inputEndDate" id="inputEndDate" readonly>
+              <label for="inputEndDate">End Date</label>
+              <input type="date" class="form-control" name="inputEndDate" 
+                id="inputEndDate" readonly>
             </div>
```

Comment:
I'd argue you should hide the end-date input entirely from the UI until multi-day trips are ready. And maybe rename start-date to 'day of travel' or similar. You and I know this projects MVP deals just with a single days itinerary but a user who doesn't know that might be confused why they can't change the end date.

Reply:
Done!